helm-chart/kuberay-operator/crds/ray.io_rayjobs.yaml

```diff
@@ -6355,11 +6355,15 @@ spec:
                             them by name
                           type: string
                         maxReplicas:
-                          description: MaxReplicas defaults to maxInt32
```

Comment:
Should we leave this comment in?

Reply:
I have added the comment in `raycluster_types.go`. However, it will not be displayed in the CRD due to length limitations. A follow-up PR will be opened to address the issue with CRD length.

Reply:
Sounds good, thanks! Not a high priority